docs/.vitepress/config.mts

```diff
@@ -1,58 +1,65 @@
-import { defineConfig } from "vitepress";
-import { presetUno } from "unocss";
-import { presetRadix, radixColors } from "../../src";
-import unocss from "unocss/vite";
+import { defineConfig } from 'vitepress';
+import { presetUno } from 'unocss';
+import { presetRadix } from '../../src';
+import unocss from 'unocss/vite';
+import { RADIX_HUES } from '../../src/consts';
 
 // https://vitepress.dev/reference/site-config
 export default defineConfig({
-  title: "unocss-preset-radix",
-  base: "/unocss-preset-radix/",
+  title: 'unocss-preset-radix',
+  base: '/unocss-preset-radix/',
   vite: {
     plugins: [
       unocss({
         presets: [
           presetUno(),
           presetRadix({
-            palette: radixColors,
-            darkSelector: ".dark",
-
+            darkSelector: '.dark',
+            lightSelector: ':root, .light',
             aliases: {
-              testa: "red",
-              testb: "amber",
+              danger: 'red',
+              warning: 'amber',
+              accent: 'blue',
             },
+            safelist: [...RADIX_HUES, 'danger', 'warning3', 'warning5A', 'warning-fg'],
           }),
         ],
         safelist: [
-          ...radixColors.map((p) => `hue-${p}`),
-          ...Array.from({ length: 12 }, (_, i) => `bg-hue${i + 1}`),
-          ...Array.from({ length: 12 }, (_, i) => `bg-hue${i + 1}A`),
+          ...['', 'A']
+            .map((alpha) =>
+              [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12].map((step) => RADIX_HUES.map((hue) => `bg-${hue}${step}${alpha}`))
+            )
+            .flat()
+            .flat(),
+          ...RADIX_HUES.map((h) => [`text-${h}9`, `text-${h}-fg`,  `text-${h}12`,  `text-${h}1`]).flat(),
```

Comment:
Pretty sure if you reverse the order of these two maps you can get rid of the `flat()` calls. (colors -> alphas)